### PR TITLE
cargo-hakari: init at 0.9.14

### PR DESCRIPTION
**Pull request by macalinao**
###### Description of changes

Adds cargo-hakari 0.9.14.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by wegank** [comment]
Have you added yourself to [maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix)?

**Comment by macalinao** [comment]
> Have you added yourself to [maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix)?

done here https://github.com/NixOS/nixpkgs/pull/188841

File: pkgs/development/tools/rust/cargo-hakari/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      cargo hakari is a command-line application to manage workspace-hack crates.
+      Use it to speed up local cargo build and cargo check commands by 15-95%,
+      and cumulatively by 20-25% or more.
+    '';
+    homepage = "https://crates.io/crates/cargo-hakari";
+
+    license = with licenses; [ mit asl20 ];
+    maintainers = with maintainers; [ macalinao ];
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/cargo-hakari/default.nix:27]:
```suggestion
    homepage = "https://crates.io/crates/cargo-hakari";
    license = with licenses; [ mit asl20 ];
    maintainers = with maintainers; [ macalinao ];
  };
```

File: pkgs/development/tools/rust/cargo-hakari/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-C4UBvxGZDpGfYokTzHQNkUkZqBNuKbE4pzOJ04sTDoY=";
+  };
+
+  cargoHash = "sha256-eQrRBmlP206MKDlXxcJ64jD6/6mv3V/sv9TsybIx+8Q=";
+
+  meta = with lib; {
+    description =
+      "Manage workspace-hack packages to speed up builds in large workspaces.";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/cargo-hakari/default.nix:16]:
```suggestion
    description = "Manage workspace-hack packages to speed up builds in large workspaces.";
```

**Pull request merged**